### PR TITLE
[12.0][FIX]shopinvader, fix bind products and bind categories buttons

### PR DESCRIPTION
**Pull request by hparfr**
In shopinvader.backend form view, "bind products" button and "bind categories" button

### CODE REVIEW

File: shopinvader/wizards/shopinvader_category_binding_wizard.py [hunk 1]
```diff
@@ -34,18 +34,15 @@ class ShopinvaderCategoryBindingWizard(models.TransientModel):
 
     @api.model
     def default_get(self, fields_list):
-        result = super(ShopinvaderCategoryBindingWizard, self).default_get(
+        res = super(ShopinvaderCategoryBindingWizard, self).default_get(
             fields_list
         )
-        backend_id = self.env.context.get("active_id")
+        backend_id = self.backend_id.browse(
```
Comment (simahawk) [inline shopinvader/wizards/shopinvader_category_binding_wizard.py]: `s/backend_id/backend` as it is a record and not and ID

**Review by simahawk — CHANGES_REQUESTED** [review]

File: shopinvader/wizards/shopinvader_category_binding_wizard.py (hunk 1 above)
Comment (hparfr) [reply shopinvader/wizards/shopinvader_category_binding_wizard.py]: As far as I know, mixing ids and records within *_id is everywhere in odoo / OCA.
Is it a new convention to stop doing it ?

Comment (simahawk) [reply shopinvader/wizards/shopinvader_category_binding_wizard.py]: For me, the OCA convention is: we try to do it better than before.
Every time I see a variable named `_id` when it's a record I :tableflip: :smile: 
So when possible, let's try to keep it clean. 
In any case, I think that it's just a bad inheritance from the days before new api came, whereas you had IDs all around and they simply did not change var names.

**Review by acsonefho — COMMENTED** [review]
LGTM!

**Review by PierrickBrun — APPROVED** [review]

**Review by Cedric-Pigeon — APPROVED** [review]
Code review

**Comment by hparfr** [comment]
@simahawk are you still requesting changes ?

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@hparfr one more thing: fix the build ;)
While you are there please squash :)

**Pull request merged**